File: android/src/main/java/com/getcapacitor/community/classes/results/SendResult.java
package com.getcapacitor.community.classes.results;

import com.getcapacitor.JSObject;
import com.getcapacitor.community.interfaces.Result;
import java.util.UUID;

public class SendResult implements Result {

    private final UUID messageID;

    public SendResult(UUID messageID) {
        this.messageID = messageID;
    }

    @Override
    public JSObject toJSObject() {
        JSObject result = new JSObject();

        if (messageID != null) result.put("messageID", messageID);

        return result;
    }
}

File: ios/Sources/BridgefyPlugin/Classes/Results/SendResult.swift
import Foundation
import Capacitor

@objc public class SendResult: NSObject, Result {
    let messageID: UUID?

    init(_ messageID: UUID?) {
        self.messageID = messageID
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()

        if let messageID = messageID {
            result["messageID"] = messageID.uuidString.lowercased()
        }

        return result as AnyObject
    }
}
